cmd/utils/flags.go

```diff
@@ -1954,8 +1954,7 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 	if ctx.IsSet(TransactionHistoryFlag.Name) {
 		cfg.TransactionHistory = ctx.Uint64(TransactionHistoryFlag.Name)
 	} else if ctx.IsSet(TxLookupLimitFlag.Name) {
-		log.Warn("The flag --txlookuplimit is deprecated and will be removed, please use --history.transactions")
-		cfg.TransactionHistory = ctx.Uint64(TransactionHistoryFlag.Name)
+		Fatalf("The flag --txlookuplimit is deprecated and may cause unexpected performance degradation issues. Please use --history.transactions instead.")
```

Comment:
use log.Crit?
if use `Fatalf`, we may lose the log
for example, run in a docker without a terminal

Reply:
should line 1951-1952 also be modified?
they handle the cases when TxLookupLimit is configured in config file